lib/pacemaker/pcmk_sched_notif.c

```diff
@@ -863,6 +864,23 @@ create_notifications(pe_resource_t * rsc, notify_data_t * n_data, pe_working_set
     }
 }
 
+/*!
+ * \internal
+ * \brief Create notification data and actions for a clone
+ *
+ * \param[in] rsc     Clone resource that notification is for
+ * \param[in] n_data  Clone notification data for some action
+ */
+void
+pcmk__create_notifications(pe_resource_t *rsc, notify_data_t *n_data)
+{
+    if (n_data != NULL) {
+        collect_notification_data(rsc, TRUE, TRUE, n_data);
+        pcmk__create_notification_keys(rsc, n_data, rsc->cluster);
+        create_notifications(rsc, n_data, rsc->cluster);
+    }
+}
```

Comment:
In general, I like functions like this to test for NULL and return if so right off, so there's less indentation later.

Reply:
That does tend to be more readable in most cases